tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts

```diff
@@ -215,11 +215,8 @@ export default {
                 InvalidIdentity: "Null",
                 WrongWeb2Handle: "Null",
                 UnexpectedMessage: "Null",
-                WrongSignatureType: "Null",
-                VerifySubstrateSignatureFailed: "Null",
-                VerifyEvmSignatureFailed: "Null",
-                RecoverEvmAddressFailed: "Null",
-                Web3NetworkOutOfBounds: "Null",
+                __Unused_WrongSignatureType: "Null",
```

Comment:
Can we use same naming format here?

Reply:
what do you mean by " naming format "? naming pattern? for example?

Reply:
I mean is to use camelCase: `UnusedWrongSignatureType`

Reply:
I think he wanted to emphasis on it.